website/docs/r/batch_pool.html.markdown

```diff
@@ -252,6 +252,8 @@ To provision a Custom Image, the following fields are applicable:
 
 A `fixed_scale` block supports the following:
 
+* `node_deallocation_option` - (Optional) Write-only property. It determines what to do with a node and its running task(s) if the pool size is decreasing. Values are `Requeue`, `RetainedData`, `TaskCompletion` and `Terminate`.
```

Comment:
we won't need to document this is write only - users don't need to know this

Reply:
Shall I just remove this property by your opinion

Reply:
As I understanding it right, that property can be set, but not retrieved? In that case, it should be mentioned in the docs for resource and be part of the state, as if you change it - it should still be part of an Update call to change the value.

Reply:
you just pull the value across in read by doing a d.get() - and please open a bug on the rest API specs and link it here, the API should be returning this value

Reply:
Done